### PR TITLE
MNT Compatibility with webonyx/graphql-php v14

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10250

### CODE REVIEW

File: tests/GraphQL/Legacy/SortBlockMutationCreatorTest.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace DNADesign\Elemental\Tests\Legacy\GraphQL;
 
 use DNADesign\Elemental\GraphQL\SortBlockMutationCreator;
+use DNADesign\Elemental\Tests\GraphQL\FakeResolveInfo;
 use DNADesign\Elemental\Tests\Src\TestElement;
 use GraphQL\Type\Definition\ResolveInfo;
```
Comment (GuySartorelli) [inline tests/GraphQL/Legacy/SortBlockMutationCreatorTest.php:8]:
```suggestion
```

**Comment by GuySartorelli** [comment]
In general I'm happy with this - there's just the outstanding question of whether to use the `FakeResolveInfo` class which is newly declared in silverstripe/silverstripe-graphql instead of the one in this module. I can't see any reason to keep the one from this module.

**Comment by emteknetnz** [comment]
I don't think we should refactor anymore than the absolute minimum to get this working given where graphql3 is in its lifecycle.

**Review by GuySartorelli — APPROVED** [review]
That's fair. I'm happy with this then.

**Pull request merged**